### ISSUE TITLE
Error when deploying meteor

### ISSUE THREAD

**Issue by ccioloca** [vercel/vercel#418]
```
⢈⠩ [METEOR-NOW] - building meteor app (this can take several minutes)Possibly unhandled Error: 1
    at Promise$_rejecter (/usr/local/lib/node_modules/meteor-now/node_modules/promise-spawner/node_modules/bluebird/js/main/promise.js:590:44)
    at /usr/local/lib/node_modules/meteor-now/node_modules/promise-spawner/index.js:85:13
    at tryCatch1 (/usr/local/lib/node_modules/meteor-now/node_modules/promise-spawner/node_modules/bluebird/js/main/util.js:63:19)
    at Promise$_callHandler [as _callHandler] (/usr/local/lib/node_modules/meteor-now/node_modules/promise-spawner/node_modules/bluebird/js/main/promise.js:695:13)
    at Promise$_settlePromiseFromHandler [as _settlePromiseFromHandler] (/usr/local/lib/node_modules/meteor-now/node_modules/promise-spawner/node_modules/bluebird/js/main/promise.js:711:18)
    at Promise$_settlePromiseAt [as _settlePromiseAt] (/usr/local/lib/node_modules/meteor-now/node_modules/promise-spawner/node_modules/bluebird/js/main/promise.js:868:14)
    at Promise$_settlePromises [as _settlePromises] (/usr/local/lib/node_modules/meteor-now/node_modules/promise-spawner/node_modules/bluebird/js/main/promise.js:1006:14)
    at Promise$_rejectPromises [as _rejectPromises] (/usr/local/lib/node_modules/meteor-now/node_modules/promise-spawner/node_modules/bluebird/js/main/promise.js:999:10)
    at Async$_consumeFunctionBuffer [as _consumeFunctionBuffer] (/usr/local/lib/node_modules/meteor-now/node_modules/promise-spawner/node_modules/bluebird/js/main/async.js:74:12)
    at Async$consumeFunctionBuffer (/usr/local/lib/node_modules/meteor-now/node_modules/promise-spawner/node_modules/bluebird/js/main/async.js:37:14)
```

Does anyone know what could be the problem?

**Comment by leo** [comment]
This doesn't look like an issue with our platform, but rather with your code. However, you can ask the awesome people at [zeit.chat](https://zeit.chat) for help, if you want.

**Comment by jamo** [comment]
(this might be correct repo: https://github.com/jkrup/meteor-now )

**Comment by ghost** [comment]
@ccioloca I have a similar issue. [I have marked it here, Please post here if you have a fix for the build issue](https://github.com/jkrup/meteor-now/issues/47).

**Comment by rim92** [comment]
I resolved the problem by migrating to version 1.2.4.3 and using the command meteor-now (ROOT_URL+MONGO_URL) with the option -d

**Comment by freedomson** [comment]
Do you have meteor installed?
